setup.py

```diff
@@ -78,7 +66,7 @@ def get_install_requires():
         'Topic :: System :: Networking',
     ],
     install_requires=get_install_requires(),
-    extras_require=extras_require,
     test_suite='nose2.collector.collector',
     scripts=['bin/netjsonconfig'],
-)
+
+)
```

Comment:
ensure the file ends with a new line and configure your editor to always add new lines at the end of files

Reply:
Okay :smile: